tasks_management/validation.py

```diff
@@ -78,40 +80,29 @@ def validate_task_executor(data, uuid=None):
     ]
 
 
-def validate_task_status(data):
-    uuid = data.get('id', None)
+def validate_task_status(uuid):
     instance = Task.objects.get(id=uuid)
     instance_status = instance.status
-    if instance_status == Task.Status.COMPLETED or instance_status == Task.Status.FAILED:
-        return [{"message": _("tasks_management.validation.task.updating_completed_task" % {
-            'status': instance_status
-        })}]
+    if instance_status in [Task.Status.COMPLETED, Task.Status.FAILED]:
+        return [
+            {"message": _("tasks_management.validation.task.updating_completed_task") % {'status': instance_status}}]
     return []
 
 
 def validate_user_exists(user_id):
-    if not User.objects.filter(id=user_id).first():
-        return [{"message": _("tasks_management.validation.group_executor.user_does_not_exist" % {
-            'code': user_id
-        })}]
-    return []
-
-
-def validate_bf_unique_code(code, uuid=None):
-    instance = TaskGroup.objects.filter(code=code, is_deleted=False).first()
-    if instance and instance.uuid != uuid:
-        return [{"message": _("tasks_management.validation.task_group.code_exists" % {
-            'code': code
-        })}]
+    if not User.objects.filter(id=user_id).exists():
```

Comment:
not working with mssql , this should work as well
```suggestion
    if not User.objects.filter(id=user_id):
```